css-view-transitions-1/Overview.bs

```diff
@@ -1649,7 +1653,8 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 
 			Note: This transform is applied to the snapshot using the `transform` property of the associated ''::view-transition-group'' pseudo-element.
 
-		* Effects on the element, such as 'opacity' and 'filter' are applied to the capture.
+		* Effects applied on the element and its descendants, such as 'opacity' and 'filter', are applied to the capture.
+			Effects applied to the element from its ancestors are ignored.
```

Comment:
Where is “Effects” as used here defined?

Reply:
That is a good question. Is there a definition in CSS for visual effects on an element which affect its descendants? The list [here](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_positioned_layout/Understanding_z-index/Stacking_context#:~:text=Element%20with%20any%20of%20the%20following%20properties%20with%20value%20other%20than%20none) is the closest I've seen in documentation. I can also just add all these properties explicitly but nicer if we can have one definition to cross-reference.